packages/adaptive_scaffold/pubspec.yaml

```diff
@@ -0,0 +1,56 @@
+name: adaptive_scaffold
+description: This package contains some helper widgets that make the process of developing adaptive layouts easier, especially with navigational elements.
+version: 0.0.1
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3Aadaptive_scaffold
+repository: https://github.com/flutter/packages/adaptive_scaffold
```

Comment:
This should be `https://github.com/flutter/packages/tree/main/packages/adaptive_scaffold`.